android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.swmansion.gesturehandler.react">
+    package="com.swmansion.gesturehandler">
```

Comment:
IIRC it's due to a bug in codegen where it ignores the set package name. Because of that the `RNGestureHandlerPackage` was moved from `com.swmansion.gesturehandler.react` to `com.swmansion.gesturehandler`. This change makes it work on Paper.

Reply:
To be precise, it looks like currently `codegenJavaPackageName` in `build.gradle` is never used. In particular, it does not seem to affect the package name of `RNGestureHandlerPackage` import when generating `PackageList.java` which is a part of auto-linking mechanism.